BAZProject/Views/MovieDetail/CollectionViewCells/RecommendationsCollectionViewCell.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  RecommendationsCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by Mario Arceo on 01/03/23.
+//
+
+import UIKit
+
+class RecommendationsCollectionViewCell: UICollectionViewCell {
+    
+    let identifier = "RecommendationCustomCell"
```

Comment:
Para no crear una instancia de esta clase es mejor poner el identifier con constante estática.
```suggestion
    static let identifier = "RecommendationCustomCell"
```